multimodal/src/autogluon/multimodal/learners/base.py

```diff
@@ -1702,6 +1703,7 @@ def predict_per_run(
             predict_data=data,
             is_train=False,
         )
+        num_gpus, strategy = run_ddp_only_once(num_gpus, strategy)
```

Comment:
Need to remove this line considering the master branch already incorporate it in the `get_num_gpus_and_strategy` class method.